dist/angular-dynamic-layout.js

```diff
@@ -608,6 +619,26 @@
         items[i].x = itemColumnsAndPosition.position.x;
         items[i].y = itemColumnsAndPosition.position.y;
       }
+
+      setDynamicLayoutHeight(columns);
+    }
+
+    /*
+     * Get the columns and get the lowest items with there heights.
+     * Then get biggest item position and set it as dynamic-layout height
+     */
+    function setDynamicLayoutHeight(columns) {
+      var i;
+      var j;
+      var dlHeight = [];
+
+      for (i = 0; i < columns.length; ++i) {
+        for (j = 0; j < columns[i].length; ++j) {
```

Comment:
This loop is not necessary as the height of the column only depends on its last element